scio-smb/src/main/java/org/apache/beam/sdk/extensions/smb/TensorFlowBucketIO.java

```diff
@@ -58,7 +58,6 @@ public static Read read(TupleTag<Example> tupleTag) {
    */
   public static <K> Write<K> write(Class<K> keyClass, String keyField) {
```

Comment:
I guess another option would be to require `numBuckets` in the Write constructor, so it isn't possible to create a Write w/o numBuckets:

```
public static <K> Write<K> write(Class<K> keyClass, String keyField, int numBuckets)
```

wdyt?

Reply:
I thought about that! but I didn't want to break the API too much, and also keep it consistent with other params (such as the `outputDirectory` param which is implemented the same way).